app/wizards/placements/edit_placement_wizard/provider_step.rb

```diff
@@ -0,0 +1,15 @@
+class Placements::EditPlacementWizard::ProviderStep < Placements::BaseStep
+  attribute :provider_id
+
+  validates :provider_id, inclusion: { in: ->(step) { step.providers_for_selection.ids } }, if: -> { provider_id != "on" }
+
+  delegate :school, to: :wizard
+
+  def providers_for_selection
+    school.partner_providers
+  end
+
+  def provider
+    @provider ||= school.partner_providers.find(provider_id).becomes(Placements::Provider) if provider_id.present? && provider_id != "on"
```

Comment:
What's the significance of `"on"` here? What does it represent/mean?

Reply:
See https://github.com/DFE-Digital/itt-mentor-services/pull/835#discussion_r1677509697